Assets/_Project/Scripts/PathData/ExerciseSubcategory.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PolSl.UrbanHealthPath.PathData
+{
+    public enum ExerciseSubcategory
```

Comment:
I don't know whether these subcatorogries are corrent. We need to get a list of exercises to figure it out I guess (?)

Reply:
Good point. However, updating them isn't big of a deal, so I'll merge current changes anyway.